Automattic/jetpack
# Site endpoint: Add is_commercial to options

Fixes: #32617 

## Proposed changes:

Added `is_commercial` property to /sites/{siteId} endpoint (inside options attribute), so Calypso can get whether a site is commercial or not and determine whether to allow the purchase of Free / Personal products.

The option is filled on wpcom: fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Swrgcnpx%2Qfvgr%2Qcebsvyrf%2Spynff.wrgcnpx%2Qfvgr%2Qcebsvyrf%2Qpbzzrepvny%2Qfgngr.cuc%3Se%3Q31qn5255%23133-og

The value of `is_commercial` has 3 possibilities:
- `true`: commercial
- `false`: non-commercial
- `null`: to be determined

### Other information:


## Jetpack product discussion
n/a

## Does this pull request change what data or activity we track or use?
no

## Testing instructions:

* Apply this PR change in your sandbox with the command: `bin/jetpack-downloader test jetpack add/is-commercial-to-options`
* Sandbox `public-api.wordpress.com`
* Open `https://developer.wordpress.com/docs/api/console/`, access the `/sites/{siteId}` wpcom v1.1
* Ensure you see `is_commercial` in the `options` object
* Please test with Jetpack self hosted, Atomic and Simple sites
* Check `/me/sites` as well
